plaidml/bridge/openvino/ops/logical_not.cpp

```diff
@@ -17,8 +17,8 @@ namespace PlaidMLPlugin {
 static OpRegistration reg("logicalnot", [](const Context& ctx) {
   IE_ASSERT(ctx.operands.size() == 1);
   auto I = ctx.operands.at(0);
-  auto T = edsl::Tensor(true);
-  auto F = edsl::Tensor(false);
+  auto T = edsl::cast(edsl::Tensor(true), DType::BOOLEAN);
+  auto F = edsl::cast(edsl::Tensor(false), DType::BOOLEAN);
```

Comment:
I'm actually puzzled by this. I think this is actually a bug in the eDSL. The principle of least surprise would say that `edsl::Tensor(true)` ought to have a element type of `DType::BOOLEAN`, but I'm guessing you had to do this because right now it thinks `true` is actually a `i1` which then considers this to be an arbitrary precision integer.

Reply:
@jbruestle WDYT?

Reply:
I would agree that Tensor(true) should be boolean.

Reply:
This whole thing can actually be replaced by the `logical_not` eDSL primitive.